icu4c/source/common/cmemory.h

```diff
@@ -779,6 +779,10 @@ class MaybeStackVector : protected MemoryPool<T, stackCapacity> {
         return this->fPool.getAlias();
     }
 
+    const T *const *getAlias() const {
```

Comment:
This new `getAlias()` method now looks fine and makes perfect sense in this context. Please squash your commits and write a commit message that describes what this now actually does.

Reply:
Done.